noir-projects/noir-protocol-circuits/crates/blob/src/blob_batching_public_inputs.nr

```diff
@@ -0,0 +1,280 @@
+use bigcurve::{
+    BigCurveTrait,
+    curves::bls12_381::{BLS12_381 as BLSPoint, BLS12_381_SCALAR_SLICES, BLS12_381Scalar},
+    ScalarFieldTrait,
+};
+use bigint::{BigNum, bignum::BigNumTrait, BLS12_381_Fq as Q, BLS12_381_Fr as F};
+use std::ops::{Add, Mul};
+use types::{
+    hash::{poseidon2_hash, sha256_to_field},
+    traits::Empty,
+    utils::{arrays::array_splice, field::{byte_to_bits_be, field_from_bytes}},
+};
+
+pub struct BatchingBlobCommitment {
+    pub point: BLSPoint,
+    pub compressed: [u8; 48], // TODO(MW): get Q::num_bytes from somewhere, rather than hardcoded 48
```

Comment:
I can't find a way to retrieve num bytes for a specific field as defined in `bignum` - am I missing something? (I don't love having a hardcoded 48 here)

Reply:
Sounds good! I'll add this